code/modules/mob/living/carbon/life.dm

```diff
@@ -149,7 +149,7 @@
 
 	else //Enough oxygen
 		failed_last_breath = 0
-		if(oxyloss)
+		if(oxyloss && (health > HEALTH_THRESHOLD_CRIT))

```

Comment:
I feel like it'd be better to support losebreath having decimals somehow

Reply:
I could just halve the bloodloss from dragging and then revisit the losebreath thing in another PR

Reply:
Would have to completely rewrite breathing for decimals to work

Reply:
I mean you'd just not lower losebreath where it's currently lowered(line 61) and instead lower it on line 111 or around there, doing damage equal to 1 or remaining losebreath

Reply:
Not sure what you mean, it bases damage on whether you missed your breath or not, not whether you have losebreath (though that does cause you to miss your breath in another proc)

Reply:
yeah I mean base the 1 damage part on how much losebreath it removes, so you can do decimals

Reply:
Oh, that makes sense yeah